app/services/collators/gross_income_collator.rb

```diff
@@ -1,6 +1,7 @@
 module Collators
   class GrossIncomeCollator
     include Transactions
+    include MonthlyEquivalentCalculatable
```

Comment:
is this a typo? Thought we were trying to remove these mixins, especially after uncovering the defect associated with one of them

Reply:
I hadn't been planning on removing this one, as it's relatively benign. But you're right, it's better that they're gone, so I've converted it to standalone class.